grazer/src/main/resources/config.properties

```diff
@@ -9,6 +9,7 @@ conceptpower.url=${conceptpower.url}
 conceptpower.concept.endpoint=Concept?id=
 concepts.type.person=http://www.digitalhps.org/types/TYPE_986a7cc9-c0c1-4720-b344-853f08c136ab
 concepts.prefix=http://www.digitalhps.org/concepts/
+conceptpower.search.endpoint = /ConceptLookup/
```

Comment:
let's use the actual search endpoint:
https://diging.atlassian.net/wiki/spaces/CODOC/pages/61152884/Search+for+a+concept